recipes/easyloggingpp/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from conans import ConanFile, CMake, tools
```

Comment:
No change in functionality.
Just to make it conform to most of the other recipes.
```suggestion
from conans import ConanFile, CMake, tools
import os
```